src/platform/cc13x2_26x2/openthread-core-cc13x2_26x2-config.h

```diff
@@ -52,7 +52,7 @@
  * Define to 1 to enable NCP UART support.
  *
  */
-#define OPENTHREAD_CONFIG_NCP_UART_ENABLE 0
+#define OPENTHREAD_CONFIG_NCP_HDLC_ENABLE 0
```

Comment:
Is this change intentional?  Comment no longer matches the #define.

Reply:
Yes it was, the token was changed in OpenThread. It still is NCP over UART, but it uses the HDLC protocol over that UART.

I will also try building with the configuration removed.

Reply:
Sure, OK.  Can we change the comment though (`@def OPENTHREAD_CONFIG_NCP_UART_ENABLE`) to match?  The mismatch makes this look like a mistake.

Reply:
Thanks!  Cleaning unnecessary things out of the config is even better.